lib/pengine/unpack.c

```diff
@@ -253,9 +253,10 @@ unpack_config(xmlNode * config, pe_working_set_t * data_set)
     data_set->stonith_timeout = (int) crm_parse_interval_spec(value);
     crm_debug("STONITH timeout: %d", data_set->stonith_timeout);
 
-    set_config_flag(data_set, "stonith-enabled", pe_flag_stonith_enabled);
+    set_config_flag(data_set, "stonith-enabled", pcmk_sched_fencing_enabled);
     crm_debug("STONITH of failed nodes is %s",
```

Comment:
going this route here is performance - right? when it happens unlikely we try to skip the outside flag-test otherwise we try to rather not have the string handling ...